definitions/variable/industry/tag_chemical_commodities.yaml

```diff
@@ -0,0 +1,23 @@
+- Chemicals Commodity:
+    - High-Value Chemicals|Ethylene:
+        description: primary chemical with molecular formula of C2H4, most important precursor for synthetic plastics
+    - High-Value Chemicals|Propylene:
+          description: primary chemical with molecular formula of C3H6, second most important precursor for synthetic plastics
+    - High-Value Chemicals|BTX:
+          description: aromatic compounds Benzene, Toluene, Xylene (BTX)
+    - Ammonia:
+        description: inorganic chemical with the formula NH3, precursor for nitrogen fertilisers
+    - Nitrogen Fertilizer:
+        description: most produced fertiliser type
+    - Methanol:
+          description: simplest alcohol used as precursor for various chemicals and fuels
+    - Formaldehyde:
```

Comment:
What's the difference between this type and resins? Can something be added to clearly distinguish this from the resins?